numba/compiler.py

```diff
@@ -613,21 +622,22 @@ def _backend(self, lowerfn, objectmode):
 
         lowered = lowerfn()
         signature = typing.signature(self.return_type, *self.args)
-        self.cr = compile_result(typing_context=self.typingctx,
-                                 target_context=self.targetctx,
-                                 entry_point=lowered.cfunc,
-                                 typing_error=self.status.fail_reason,
-                                 type_annotation=self.type_annotation,
-                                 library=self.library,
-                                 call_helper=lowered.call_helper,
-                                 signature=signature,
-                                 objectmode=objectmode,
-                                 interpmode=False,
-                                 lifted=self.lifted,
-                                 fndesc=lowered.fndesc,
-                                 environment=lowered.env,
-                                 has_dynamic_globals=lowered.has_dynamic_globals,
-                                 )
+        self.cr = compile_result(
```

Comment:
There seems to be a lot of formatting churn?! autopep8?

Reply:
All manual =)